AUTH-README.md

```diff
@@ -0,0 +1,327 @@
+# 801.1X & Captive Portal Authentication with Faucet
```

Comment:
see https://github.com/Bairdo/faucet/commit/b16e835e883742975076301189155f18f0d4fd77
for comments